simple_message/CMakeLists.txt

```diff
@@ -9,11 +9,11 @@ set(ROS_BUILD_STATIC_LIBS true)
 set(ROS_BUILD_SHARED_LIBS false)
 
 # The simple_message library is designed to cross compile on Ubuntu
-# and various robot controllers.  This requires conditionally compiling
-# certain functions and headers.  The definition below enables compiling
-# for a ROS node.
-add_definitions(-DROS=1)           #build using ROS libraries
-add_definitions(-DLINUXSOCKETS=1)  #use linux sockets for communication
+# and various robot controllers. This requires conditionally compiling
+# certain functions and headers. The default flags in the file included
+# here enable compiling for a ROS node. This file is also exported to
+# dependent packages via the `catkin_package` commmand.
+include(cmake/platform_build_flags.cmake)
```

Comment:
I think documenting this as a subsection in the README might be helpful

Reply:
I guess the [wiki page](http://wiki.ros.org/simple_message) would be best but it may be out-of-date (last edit ~3 years ago)